azext_connect/custom.py

```diff
@@ -517,11 +518,22 @@ def bind_webapp_postgres(
         scope = '/subscriptions/{0}/resourceGroups/{1}'.format(subscription, resource_group)
         source = '{0}/providers/Microsoft.Web/sites/{1}'.format(scope, appname)
         target = _get_target_id(scope, postgres=server, database=database)
-        result = _bind(
-            cmd, subscription, resource_group, name, source,
-            target, authtype, permission, client_id, client_secret, username, password
-        )
-        print(json.dumps(result, indent=2))
+        for i in range(3):
+            try:
+                result = _bind(
+                    cmd, subscription, resource_group, name, source,
+                    target, authtype, permission, client_id, client_secret, username, password
+                )
+            except Exception as e:
+                s = str(e)
+                if s.find('\"UnauthorizedResourceAccess\"') and authtype == 'Secret':
+                    print('retry left: {0}'.format(3-i))
+                    print('Username or password error, please input the correct username and password:')
```

Comment:
please help provide an option to reset the password since user may forget the password.